src/Field/Configurator/TextEditorConfigurator.php

```diff
@@ -22,7 +22,10 @@ public function supports(FieldDto $field, EntityDto $entityDto): bool
     public function configure(FieldDto $field, EntityDto $entityDto, AdminContext $context): void
     {
         if ('rtl' === $context->getI18n()->getTextDirection()) {
-            $field->addCssAsset((new AssetDto('bundles/easyadmin/field-text-editor.rtl.css')));
+            $assetDto = new AssetDto('field-text-editor.rtl.css');
+            $assetDto->setInternalEasyAdminAsset();
```

Comment:
This should use new method:
```suggestion
            $assetDto->setPackageName(AssetPackage::PACKAGE_NAME);
```